bundles/paikkatietoikkuna/admin/admin-layeranalytics/Flyout.js

```diff
@@ -35,19 +37,52 @@ Oskari.clazz.define('Oskari.framework.bundle.admin-layeranalytics.Flyout',
 
             this.updateListing();
         },
+        generateToScaleURL (details) {
+            const stack = details.stack[0];
+            let toScaleURL = '/?coord=' + stack.x + '_' + stack.y;
+            toScaleURL += '&mapLayers=';
+
+            for (const [index, value] of stack.layers.entries()) {
+                toScaleURL += value; // add layer id
+                toScaleURL += '+100'; // add layer opacity
+                toScaleURL += '+'; // add layer default style as empty string
+                if (index !== (stack.layers.length - 1)) {
+                    toScaleURL += ','; // add layer separator if not last layer in stack
+                }
+            }
+
+            return toScaleURL;
+        },
         updateListing () {
```

Comment:
Perhaps rename updateListing() to updateUI() or similar since it's no longer just the listing?

Reply:
Renamed.